### ISSUE TITLE
`./scripts/setup.sh --local` fails with test errors on v0.9.16 tag on ubuntu:bionic

### ISSUE THREAD

**Issue by vors** [facebook/pyre-check#683]
The tests are failing.

## Repro

Repro in a docker container.

Dockerfile:
```
FROM ubuntu:bionic

ARG PYRE_VERSION=0.9.16

ENV DEBIAN_FRONTEND noninteractive

RUN apt-get update \
    && apt-get install -y \
       build-essential \
       software-properties-common \
    && rm -rf /var/lib/apt/lists/*

# pyre dependencies
RUN add-apt-repository ppa:avsm/ppa \
    && apt-get update \
    && apt-get install -y \
       opam \
    && rm -rf /var/lib/apt/lists/*

WORKDIR /build

RUN git clone https://github.com/facebook/pyre-check \
    && cd pyre-check \
    && git checkout v$PYRE_VERSION

WORKDIR /build/pyre-check

RUN opam init --disable-sandboxing \
    && opam switch create 4.10.2 \
    && eval $(opam env)

# The error happens here
RUN ./scripts/setup.sh --local \
    && cd source \
    && make

```

Try to build it.

## Expected

Build should pass

## Actual

Build fails with test failure.

```
Traceback (most recent call last):
  File "./scripts/setup.py", line 334, in <module>
    setup(Setup)
  File "./scripts/setup.py", line 329, in setup
    build_type_override=parsed.build_type,
  File "./scripts/setup.py", line 250, in full_setup
    run_make("release_test" if self.release else "test")
  File "./scripts/setup.py", line 242, in run_make
    add_environment_variables=opam_environment_variables,
  File "./scripts/setup.py", line 270, in run
    env=environment_variables,
  File "/usr/lib/python3.6/subprocess.py", line 356, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['make', 'test']' returned non-zero exit status 2.
```

**Comment by kinto0** [comment]
hmm... I can't repro this locally - I think your `PYRE_VERSION` is incorrect. Latest release I see is `0.9.11` https://github.com/facebook/pyre-check/releases

**Comment by vthemelis** [comment]
I think that this is likely due to https://github.com/facebook/pyre-check/issues/760

**Comment by WangGithubUser** [comment]
@vors It should have been fixed in c96227cbc847822f7ad4959d74914d126f964782.
